app/lib/pre_assembly/batch.rb

```diff
@@ -56,7 +56,7 @@ def run_log_msg
         project_name: project_name,
         bundle_dir: bundle_dir,
         assembly_staging_dir: Settings.assembly_staging_dir,
-        environment: ENV['RAILS_ENV']
+        environment: ENV.fetch('RAILS_ENV')
```

Comment:
This is fine, but we probably should use `Rails.env` here. That handles the case where `RAILS_ENV` is omitted and defaults to `development`.

Reply:
thanks.